webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala

```diff
@@ -840,15 +855,13 @@ class DirectiveEditForm(
     }
   }
 
-  private[this] def updateAndDisplayNotifications(): NodeSeq = {
-
+  private[this] def displayNotifications(): NodeSeq = {
     val notifications = formTracker.formErrors
-    formTracker.cleanErrors
```

Comment:
This is the faulty line, it cleans errors too early. The form is re-rendered with errors near each field, so we need to clean only after re-rendering